### ISSUE TITLE
Move CO2 emissions of biomass to Emissions section

### ISSUE THREAD

**Issue by mabijkerk** [quintel/etmodel#4325]
Currently, we have a slide called "CO2 emissions of biomass":

<img width="458" alt="Screenshot 2024-08-29 at 10 02 14" src="https://github.com/user-attachments/assets/c701a153-a766-47e2-bafe-bb93cd6d0779">

This slides lets a user choose whether to include biogenic emissions in the ETM calculations or not. This slide is located in the Supply section, under Biomass. I propose to move it to the Emissions sections, though I'm not sure yet where exactly to put it. Perhaps under Emission factors.

Any thoughts @kaskranenburgQ @kndehaan @Charlottevm?

**Comment by Charlottevm** [comment]
Good idea @mabijkerk , I could also live with a new header (eg 'biomass') as it feels slightly different from emission factors and deserves its own section as it is quite an important topic. But if you all feel it should be under emission factors I'm also fine with that, no strong opinion here.
